doc/user-guide/running.md

```diff
@@ -129,3 +139,34 @@ _ = execute_notebook(
     progress_bar=False,
 )
 ```
+
+## Removing cells
+
+```{versionadded} 0.0.21
+```
+
+```{admonition} Command-line equivalent
+:class: dropdown
+
+`ploomber-engine nb.ipynb output.ipynb --remove-tagged-cells remove`
+```
+
+If there are notebooks you want to remove before execution, tag them and use `remove_tagged_cells`. [This sample notebook](https://github.com/ploomber/ploomber-engine/blob/main/examples/remove.ipynb) contains one cell that will fail, if executed; however, the cell contains the tag `"remove"`, so let's remove it before execution:
+
+```{code-cell} ipython3
+:tags: [hide-output]
+
+%%sh
+curl https://raw.githubusercontent.com/ploomber/ploomber-engine/main/examples/remove.ipynb \
+    --output running-remove.ipynb
+```
+
+```{code-cell} ipython3
+:tags: [hide-output]
+
+_ = execute_notebook(
+    "running-remove.ipynb",
+    output_path=None,
+    remove_tagged_cells=["remove"],
```

Comment:
A cell is tagged `remove_another` and added like `remove_tagged_cells=["remove_another"] - Does this work?